pkgs/development/python-modules/awacs/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+
+  # python dependencies
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "awacs";
+  version = "2.2.0";
+
+  disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```
https://github.com/cloudtools/awacs/blob/2.2.0/setup.cfg#L33